CovidShield/mobile
# Fix Android EN bridge

This PR fixes Android ExposureNotification.start doesn't handle `startResolutionForResult`.

Demo video https://drive.google.com/file/d/1QPR_z0houVS8-O7Haq9TIc2hdCN_j19H/view?usp=sharing

Resolves https://github.com/CovidShield/mobile/issues/115

## How to test?
- Run react native packager `yarn start`.
- In terminal, make sure you one device connected to computer, run `adb reverse tcp:8081 tcp:8081`.
  + If you have issue, run `adb kill-server && adb start-server`.
- Open project in Android Studio and build to real device. 
- Go through onboarding flow. 
- Expect to see Exposure Notification dialog.
